### ISSUE TITLE
Authentication Breaks on Localized Routes

### ISSUE THREAD

**Issue by liam-milbel** [codezero-be/laravel-localized-routes#62]
For the website I'm working on, the vast majority of routes will be localized. All of the admin area routes are broken with localized routes except for the dashboard. For example, there is a `/admin/images` route but whenever I visit it, I get logged out and redirected back to the login page.
None of the `/admin/*` routes work at all.

This happens both when the locale prefix is present and when it's omitted.

Obviously this shouldn't happen. What can I do to get this working as normal?

**Comment by ivanvermeyen** [comment]
Hi,

Can you show me the routes?

**Comment by liam-milbel** [comment]
We're using laravel-modules so this doesn't cover all of the routes but it does cover the generic routes and the auth routes.

https://gist.github.com/liam-milbel/361d9feefa3f19fc379335eb7cbed829

**Comment by ivanvermeyen** [comment]
The only thing the `Route::localized()` macro does is wrap the function in a group with a locale prefix and name prefix.
The rest is just default Laravel routing.

I did see you have some duplicate routes in the admin and user group, but that probably doesn't matter for this.
It's hard to debug tho without seeing the bigger picture.
I would take a step back, disable all routes and add them in stages to find where it goes wrong...

**Comment by liam-milbel** [comment]
I was able to figure this out. The routes I had linked previous all worked (I didn't realise at the time). The routes that didn't work were those defined in modules ([nWidart/laravel-modules](https://github.com/nWidart/laravel-modules)), specifically modules made back in Laravel 5.
Upgrading to Laravel 6, there is a migration command to correct these modules however it seems to miss this aspect of routes.
Now that I'm aware of the problem, I am able to fix it.
